tests/unit/states/test_x509.py

```diff
@@ -0,0 +1,32 @@
+from salt.exceptions import SaltInvocationError
+from tests.support.mock import MagicMock, call, patch
+from tests.support.unit import TestCase, skipIf
+
+try:
+    from salt.states import x509
+except ImportError:
+    x509 = None
+
+
+@skipIf(x509 is None, "X509 cant be imported!")
+class X509TestCase(TestCase):
+    def test_deprecated_managed_private_key(self):
```

Comment:
In this case I would rather not include a test case. I do not think this adds value to the test suite down the road, since it is just testing code was removed that we removed.